src/Options/Applicative/Complicated.hs

```diff
@@ -20,12 +21,15 @@ import           Options.Applicative.Types
 import           Options.Applicative.Builder.Extra
 import           Options.Applicative.Builder.Internal
 import           Stack.Prelude
+import           Stack.Types.Config
 import           System.Environment
 
+type AddCommand env =
```

Comment:
I wondered if `Stack.Types.Config` would be a better place for the type synonym. It is imported by `Main` and it already exports some types related to commands.